FirebaseCoreSDK/Firebase/Database/IFirebaseDatabase.cs

```diff
@@ -9,6 +9,6 @@
 
     public interface IFirebaseDatabase
     {
-        IDatabaseRef Ref(string path);
+        IDatabaseRef Ref(string path, QueryBuilder queryBuilder = null);
```

Comment:
it's better to create an overload without any optional params gere

Reply:
Added Overload instead of Optional parameter